### PR TITLE
Fix build script

### PR DESCRIPTION
**Pull request by eneufeld**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -44,16 +44,51 @@ pipeline {
...
+                        }
+                    }
+                }
+            }
+        }
+        stage('Copy server') {
+            steps {
+                container('ci') {
+                    timeout(30){
+                        dir('.') {
```
Comment (lucas-koehler) [inline Jenkinsfile:62]: is this necessary? I thought this is basically the default but maybe it's different when building with docker and kubernetes.

File: Jenkinsfile [hunk 2]
```diff
@@ -44,16 +44,51 @@ pipeline {
...
+                        }
+                    }
+                }
+            }
+        }
         stage('Build client') {
             steps {
                 container('ci') {
                     timeout(30){
                         dir('.') {
```
Comment (lucas-koehler) [inline Jenkinsfile:73]: same as above

**Review by lucas-koehler — COMMENTED** [review]
LGTM. I only have 2 small comments about two potentially unnecessary configs ;)

File: Jenkinsfile (hunk 1 above)
Comment (eneufeld) [reply Jenkinsfile:62]: I don't know, but at least it is explicit ;-)

Comment (lucas-koehler) [reply Jenkinsfile:62]: Alright, fine with me :D

**Review by lucas-koehler — APPROVED** [review]

**Pull request merged**